### PR TITLE
Add storkctl test for listing snapshots, with no snapshots available

### PR DESCRIPTION
**Pull request by rodrigc**

### CODE REVIEW

**Comment by disrani-px** [comment]
@rodrigc Can you rebase your changes, with one commit for the dependencies and one commit for the test files. We can merge it then

File: pkg/storkctl/fake_api_server_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+func objBody(codec runtime.Codec, obj runtime.Object) io.ReadCloser {
+	return ioutil.NopCloser(bytes.NewReader([]byte(runtime.EncodeOrDie(codec, obj))))
+}
+
+func stringBody(body string) io.ReadCloser {
+	return ioutil.NopCloser(bytes.NewReader([]byte(body)))
+}
+
+type FakeResources struct {
```
Comment (disrani-px) [inline pkg/storkctl/fake_api_server_test.go]: Not being used from here to the end of this file

File: pkg/storkctl/fake_api_server_test.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	"k8s.io/kube-openapi/pkg/util/proto"
+
+	"k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/resource"
+	"k8s.io/kubernetes/pkg/kubectl/scheme"
+)
+
+var openapiSchemaPath = filepath.Join("..", "..", "..", "..", "api", "openapi-spec", "swagger.json")
+
+var unstructuredSerializer = resource.UnstructuredPlusDefaultContentConfig().NegotiatedSerializer
```
[the remark this answers is not in the document]
Comment (rodrigc) [reply pkg/storkctl/fake_api_server_test.go]: `unstructuredSerializer` is used in `storkctl_test.go`

**Comment by disrani-px** [comment]
Codecov is complaining because we actually added tests (and actually improved coverage). Ok to merge.

**Pull request merged**